javascript-forms/form-values.js

```diff
@@ -0,0 +1,10 @@
+const form = document.querySelector('form');
+
+function logSubmit(event) {
+  event.preventDefault();
+  const myFormData = new FormData(event.target);
+  const formDataObj = {};
+  myFormData.forEach((value, key) => (formDataObj[key] = value));
+  console.log(formDataObj);
```

Comment:
you shouldn't be using classes here yet. 
The instructions say on #12 to get the value of each of your form controls by name via the forms `elements` property... 
To start replace lines 5-8 with `console.log(form.elements)` to see the `elements` object. It should look like this:

![Screenshot 2023-05-14 at 2 41 05 PM](https://github.com/annaaksenov/m1-0323-code-solutions/assets/7886753/78737093-3e02-40b6-85cc-5e142eb5b970)

You'll notice the `email`, `name`, and `message` properties which in turn also have their own `value` property. If you save `form.elements.email.value` to a variable, you should be able to see what it was when the form was submitted, and you can do the same for the other properties.

Then you can take those stored values and assign them to an object that you log in step #13.